### PR TITLE
pihole collector: add alarms

### PR DESCRIPTION
**Pull request by ilyam8**
##### Summary
Alarm for [`pihole`](https://github.com/netdata/go.d.plugin/tree/master/modules/pihole) collector.

Arlarms to monitor:
 - Pi-hole availability
 - blocked dns queries precentage
 - blocklist last update time
 - gravity.list file existence
 - Pi-hole blocking functionality status (on/off)

Related: #6204

##### Component Name
[health/](https://github.com/netdata/netdata/tree/master/health)

##### Additional Information

Test:
 - i installed the branch
 - checked the dashboard alarm section

Alarms are there.

### CODE REVIEW

**Comment by ilyam8** [comment]
@modem7 i think we are wrong on `blocklist last update` alarm. Yes, the cron job runs every week, but it does not guarantee the list is updated on every run.

So we need to understand if there is an update schedule or no. I hope devs will give some info about it. 

Am i right?

**Comment by modem7** [comment]
Unsure myself, but I would have assume that the counter is reset whenever the cronjob runs, but without knowing a bit more of the code, I couldn't be 100% on that!

**Comment by ilyam8** [comment]
> Unsure myself, but I would have assumed that the counter is reset whenever the cronjob runs, but without knowing a bit more of the code, I couldn't be 100% on that!

could you run the job by hand and see if the counter resets?

**Comment by modem7** [comment]
> 
> 
> > Unsure myself, but I would have assumed that the counter is reset whenever the cronjob runs, but without knowing a bit more of the code, I couldn't be 100% on that!
> 
> could you run the job by hand and see if the counter is reset?

OK, so I ran pihole -g which I can only assume is the same command the cronjob runs, and the timer has been reset to 0 as hoped!

**Comment by ilyam8** [comment]
@modem7

I just realized that I could do it myself :see_no_evil: 	

> pihole -g which I can only assume is the same command the cronjob runs

Same, you are right.

File: health/health.d/pihole.conf [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  # Blocked DNS queries.
+
+ template: pihole_blocked_queries
+       on: pihole.dns_queries_percentage
+    every: 10s
+    units: %
+     calc: $blocked
+     warn: $this > ( ($status >= $WARNING ) ? ( 50 ) : ( 75 ) )
+     crit: $this > ( ($status >= $CRITICAL) ? ( 75 ) : ( 85 ) )
```
Comment (paulkatsoulakis) [inline health/health.d/pihole.conf]: Shouldn't critical be all the way to 99?
I may miss how exactly the rules are parsed

**Pull request merged**